cores/arduino/stm32/stm32_eeprom.h

```diff
@@ -80,7 +80,21 @@ extern "C" {
  */
 #define FLASH_PAGE_SIZE     ((uint32_t)(16*1024)) /* 16kB page */
 #endif
+
+#if defined(STM32L0xx)
```

Comment:
What about STM32L1xx which have also an embedded EEPROM ? For example STM32L152RE  has 16 Kbytes of true EEPROM with ECC  but it defines `FLASH_EEPROM_BASE` and `FLASH_EEPROM_END`.

Reply:
this PR is for stm32L0, isn't

Reply:
on stm32L0x EEPROM section is named DATA_EEPROM_BASE/DATA_EEPROM_END
on stm32L1x section is named FLASH_EEPROM_BASE/FLASH_EEPROM_END

Reply:
> 
> 
> this PR is for stm32L0, isn't

Yes but issue is also for L1. And it is mentioned in the issue by the submitter:

> When I tested the same function on Nucleo L152RE, which should have 16KB EEPROM, I got 256.